context/app/static/js/components/Detail/DetailLayout/DetailLayout.jsx

```diff
@@ -1,32 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import useEntityStore from 'js/stores/useEntityStore';
+import useEntityStore, { savedAlertStatus, editedAlertStatus } from 'js/stores/useEntityStore';
 import TableOfContents from '../TableOfContents';
 import { Content, FlexRow, StyledAlert } from './style';
 import { getSections } from './utils';
 
 const entityStoreSelector = (state) => ({
-  shouldDisplaySavedListsAlert: state.shouldDisplaySavedListsAlert,
-  toggleShouldDisplaySavedListsAlert: state.toggleShouldDisplaySavedListsAlert,
+  shouldDisplaySavedOrEditedAlert: state.shouldDisplaySavedOrEditedAlert,
+  setShouldDisplaySavedOrEditedAlert: state.setShouldDisplaySavedOrEditedAlert,
 });
 
 function DetailLayout(props) {
   const { sectionOrder, children } = props;
 
-  const { shouldDisplaySavedListsAlert, toggleShouldDisplaySavedListsAlert } = useEntityStore(entityStoreSelector);
+  const { shouldDisplaySavedOrEditedAlert, setShouldDisplaySavedOrEditedAlert } = useEntityStore(entityStoreSelector);
 
   // section hash must match section id in each component
   const sections = new Map(getSections(sectionOrder));
 
   return (
     <>
-      {shouldDisplaySavedListsAlert && (
-        <StyledAlert severity="success" onClose={() => toggleShouldDisplaySavedListsAlert()}>
+      {shouldDisplaySavedOrEditedAlert === savedAlertStatus && (
```

Comment:
This seems odd... Could they both be `false`... but equal to each other... so it evaluates to true?